docs/content/how-do-i-write-policies.md

```diff
@@ -1053,6 +1053,79 @@ The result:
 +-----------+------------------------+
 ```
 
+## Universal Quantification (FOR ALL)
+
+Like SQL, Rego does not have a direct way to express _universal quantification_
+("FOR ALL"). However, like SQL, you can use other language primitives (e.g.,
+[Negation](#negation)) to express FOR ALL. For example, imagine you want to
+express a policy that says (in English):
+
+```
+There must be no apps named "bitcoin-miner".
+```
+
+You may start by defining a rule named `no_bitcoin_miners` like so:
+
+```ruby
+no_bitcoin_miners {
+    app := apps[_]
+    app.name != "bitcoin-miner"  # THIS IS NOT CORRECT.
+}
+```
+
+Unfortunately this rule is incorrect. You can see this by testing the rule:
+
+```ruby
+> no_bitcoin_miners with apps as [{"name": "bitcoin-miner"}, {"name": "web"}]
+true
+```
+
+The reason the rule is incorrect is that variables in Rego are _existentially
+quantified_. This means that rule bodies and queries express FOR ANY and not FOR
+ALL. To express FOR ALL in Rego use [Negation](#negation) and a bit of basic
+logic:
+
```

Comment:
How about a bridge that gives the english version before the 2 complements:

"For this policy, you define a rule that finds if there exists a bitcoin-mining app (which is easy using the `some` keyword).  And then you use negation to check that there is NO bitcoin-mining app.  Technically, you're using 2 negations and an existential quantifier, which is logically the same as a universal quanitfier."